test/Npgsql.Tests/Support/PgServerMock.cs

```diff
@@ -24,12 +25,14 @@ class PgServerMock : IDisposable
         internal int ProcessId { get; }
 
         internal PgServerMock(
+            Socket socket,
```

Comment:
Closing the stream should implicitly close the socket, since the latter is owned by the former.

Reply:
Yes and no - while it does close the socket, there is a timeout, when the socket is still sending queued data. For some reason, I thought it is the same for the read side.